core/src/main/java/com/sequenceiq/cloudbreak/service/ComponentConfigProvider.java

```diff
@@ -80,6 +81,18 @@ public AmbariDatabase getAmbariDatabase(Long stackId) {
         }
     }
 
+    public CloudbreakDetails getCloudbreakDetails(Long stackId) {
```

Comment:
yepp, but it will be used in the next pull request that relates to this JIRA:
https://hortonworks.jira.com/browse/BUG-61940